tests/test_reader.py

```diff
@@ -26,7 +26,7 @@ def test_nexus_conversion(caplog, tmp_path):
     test.check_reproducibility_of_nexus()
 
     caplog.clear()
-    dir_path_multi = Path(__file__).parent / "data_multi"
+    dir_path_multi = Path(__file__).parent / "data_multi/witec"
```

Comment:
You can parametrize the tests with the different folders here (can also set a param for `caplog`), see e.g. https://github.com/FAIRmat-NFDI/pynxtools-xps/blob/main/tests/test_reader.py#L39 or https://github.com/FAIRmat-NFDI/pynxtools-stm/blob/main/tests/test_reader.py#L13

So for you, something like (not tested for typos etc.)
```
@pytest.mark.parametrize(
    "data_dir, caplog_level"
    [
        ("rod", "ERROR"),
        ("witec", "WARNING"),
    ],
)
def test_raman_reader(data_dir, caplog_level, tmp_path, caplog):
    dir_path_multi = Path(__file__).parent / f"data_multi/{data_dir}"
    test = ReaderTest(
        nxdl="NXraman",
        reader_name="raman_multi",
        files_or_dir=glob(os.path.join(dir_path_multi, "*")),
        tmp_path=tmp_path,
        caplog=caplog,
    )
    test.convert_to_nexus(caplog_level=caplog_level, ignore_undocumented=False)
    test.check_reproducibility_of_nexus()
```